docs/providers/hp/network.md

```diff
@@ -173,3 +173,73 @@ Returns the port in the callback `f(err, port)`
 Destroys the specified port
 
 Takes port or portId as an argument  and returns the id of the destroyed port in the callback `f(err, portId)`
+
+**Security Groups**
+
+#### client.getSecurityGroups(callback)
+Lists all security groups that are available to use on your Openstack account
+
+Callback returns `f(err, securityGroups)` where `securityGroups` is an `Array`
+
+#### client.getSecurityGroup(securityGroup, callback)
+Gets specified security group
+
+Takes securityGroup or securityGroupId as an argument and returns the security group in the callback
+`f(err, securityGroup)`
+
+#### client.createSecurityGroup(options, callback)
+Creates a security group with the options specified
+
+Options are as follows:
+
+```js
+{
+  name: 'securityGroupName', // required, name of security group
+  description : 'security group description',  // optional, description of security group
+  tenantId : 'tenantId'     // optional, The ID of the tenant who owns the security group. Admin-only
+}
+```
+Returns the created security group in the callback `f(err, securityGroup)`
+
+#### client.destroySecurityGroup(securityGroup, callback)
+Destroys the specified security group
+
+Takes securityGroup or securityGroupId as an argument  and returns the id of the destroyed security group in the callback `f(err, securityGroupId)`
+
+**Security Group Rules**
+
+#### client.getSecurityGroupRules(callback)
+Lists all security group rules that are available to use on your Openstack account
+
+Callback returns `f(err, securityGroupRules)` where `securityGroupRules` is an `Array`
+
+#### client.getSecurityGroupRule(securityGroupRule, callback)
+Gets specified security group rule
+
+Takes securityGroupRule or securityGroupRuleId as an argument and returns the security group rule in the callback
+`f(err, securityGroupRule)`
+
+#### client.createSecurityGroupRule(options, callback)
+Creates a security group rule with the options specified
+
+Options are as follows:
+
+```js
+{
+  securityGroupId: 'securityGroupId', // required, The security group ID to associate with this security group rule.
+  direction: 'ingress|egress', // required, The direction in which the security group rule is applied.
+  ethertype: 'IPv4|IPv6', // optional,
```

Comment:
`ethertype` is what the underlying HTTP API uses: http://developer.openstack.org/api-ref-networking-v2.html#security_groups